tdrs-backend/tdpservice/scheduling/tasks.py

```diff
@@ -41,3 +44,34 @@ def users_to_deactivate(days):
         last_login__gte=datetime.now(tz=timezone.utc) - timedelta(days=days+1),
         account_approval_status=AccountApprovalStatusChoices.APPROVED,
         )
+
+def get_ofa_admin_user_emails():
+    """Return a list of OFA System Admin users."""
+    return [user.email for user in User.objects.filter(
+        groups='4'
+    )]
+
+def get_num_access_requests():
+    """Return the number of users requesting access."""
+    return len(User.objects.filter(
+        account_approval_status=AccountApprovalStatusChoices.ACCESS_REQUEST,
```

Comment:
should this `filter` make use of the `access_requested_date` as well? or are we just interested in counting all of the users and simply showing the request date on the admin?

Reply:
Are you thinking we only filter users that have requested access in the last 24hrs? To avoid bloat if certain users have been looked at but not approved yet since they would have been included in a previous email?

Reply:
that's what i was thinking! open to opinions on that of course

Reply:
I tend to agree with Jan on this, the email is only to notify, admin can alway go back and check the requests in the admin page. So older pending requests should be taken care of in the admin page and not by email notification.